t/t0061-run-command.sh

```diff
@@ -210,10 +210,24 @@ test_expect_success MINGW 'verify curlies are quoted properly' '
 	test_cmp expect actual
 '
 
-test_expect_success MINGW 'can spawn with argv[0] containing spaces' '
-	cp "$GIT_BUILD_DIR/t/helper/test-fake-ssh$X" ./ &&
-	test_must_fail "$PWD/test-fake-ssh$X" 2>err &&
-	grep TRASH_DIRECTORY err
+test_expect_success MINGW 'can spawn .bat with argv[0] containing spaces' '
+	bat_file="$TRASH_DIRECTORY/bat with spaces in name.bat" &&
```

Comment:
How about renaming `bat_file` to `bat`? We know it's a file (and technically, the variable holds the _path_ to that file, so it is a misnomer anyway, so why not use a shorter misnomer?).